calico_containers/adapter/datastore.py

```diff
@@ -365,12 +405,14 @@ def add_ip_pool(self, version, pool):
         # Normalize to CIDR format (i.e. 10.1.1.1/8 goes to 10.0.0.0/8)
         pool = pool.cidr
 
-        # Check if the pool exists.
-        if pool in self.get_ip_pools(version):
-            return
+        key = IP_POOL_KEY % {"version": version,
+                             "pool": str(pool).replace("/", "-")}
 
-        pool_path = IP_POOL_PATH % {"version": version}
-        self.etcd_client.write(pool_path, str(pool), append=True)
+        data = {"cidr" : str(pool)}
+        if ipip:
+            data["ipip"] = "tunl0"
```

Comment:
currently mandated by Felix - apparently its the default and is created automatically.  Shaun knows more about this.